R/deployApp.R

```diff
@@ -193,20 +210,6 @@ deployApp <- function(appDir = getwd(),
   # normalize appDir path
   appDir <- normalizePath(appDir)
 
-  # create the full path that we'll deploy (append document if requested)
-  # TODO(HW): we use appPrimaryDoc here, but we have not inferred it yet
```

Comment:
I now understand this — there's a subtle difference between deploying a document and a directory containing a single document.